docs/pct/README.md

```diff
@@ -0,0 +1,291 @@
+
+The `--list` or `-l` flag displays a list of locally available templates located in the default template location `$HOME/.pdk/pct/`. The list of templates is also available by calling `pct new` without flags.
+
+``` bash
+pct new
+pct new --list
+```
+
+Example output:
+
+```bash
+           DISPLAYNAME          |          NAME           |  TYPE
+--------------------------------+-------------------------+----------
+                                |                         |
+  Bolt Plan                     | bolt-plan               | item
+  Bolt Project                  | bolt-project            | project
+  Bolt PowerShell Task          | bolt-pwsh-task          | item
+  Bolt YAML Plan                | bolt-yaml-plan          | item
+  Puppet Module Managed Gemfile | git-attributes          | item
+  Puppet Class                  | puppet-class            | item
+  Puppet Content Template       | puppet-content-template | project
+  Puppet Defined Type           | puppet-defined-type     | item
+  Puppet Fact                   | puppet-fact             | item
+  Puppet Module                 | puppet-module           | project
+  Puppet Resource API Provider  | rsapi-provider          | item
+  Puppet Resource API Transport | puppet-transport        | item
+```
+
+Using the available templates above, its time to generate some content.
+
+``` bash
+pct new <template>
+```
+
+Replace `<template>` with the `name` of the template containing the content you want.
+
+By default the `new <template>` function will use the directory name of your current working directory to "name" your new content.
+To override this behavour use the `--name` or `-n` flag.
+
+``` bash
+pct new <template> --name MyProject
+```
+
+By default the `new <template>` function will output the template content to the current working directory.
+To override this behavour use the `--output` or `-o` flag.
+
+``` bash
+pct new <template> --output /path/to/your/project
+```
+
+> :memo: Not all templates require a `name` and providing one will have no effect on the generated content.
+
+Example usage:
+
+``` bash
+> cd /home/me/projects/MyBoltProject
+> pct new bolt-project
+
+> pct new puppet-module -n MyNewProject -o /home/me/projects/
+> cd /home/me/projects/MyNewProject
+> pct new puppet-fact -n ApplicationVersion
+> pct new rsapi-provider -n Awesomething
+> pct new puppet-transport -n AwesomethingApi
+```
+
+### Template Updates
+
+`pct new` will **NOT** update your code to a newer version of a template.
+
+If you run a `pct new` command using a `project` template, the project will replace the content within the output directory with the template code.
+
+If you run a `pct new` command using an `item` template, the item will suppliment the content within the output directory with the template code. If files / folders that are named the same as the template content already exist, it will overwite this content.
+
+## Writing Templates
+
+### Dos and Don'ts
```

Comment:
This is a great start, and something we'll add to over time. I think this section should be after the structure and example sections in this block